src/middlewared/middlewared/plugins/gluster_linux/eventsd.py

```diff
@@ -147,3 +167,71 @@ def sync(self, job):
             return json.loads(out.strip())['output']
         else:
             raise CallError(err.strip())
+
+    @job(lock=EVENTSD_LOCK)
```

Comment:
Isnt this supposed to be a private method?

Reply:
I've made it private